boards/common/frdm/Makefile.include

```diff
@@ -8,10 +8,10 @@ export DEBUG_ADAPTER ?= dap
 #export DEBUG_ADAPTER ?= jlink
 
 # OpenOCD v0.10.0 and newer have built-in support for disabling the Kinetis
-# watchdog automatically. However, current stable releases of Ubuntu and Debian
-# have only version 0.9.0 and older OpenOCD packages (Ubuntu 17.04, Debian Jessie)
-# Set this to 0 to avoid the extra manual step of disabling the watchdog.
-export USE_OLD_OPENOCD ?= 1
+# watchdog automatically. Some older releases of Ubuntu and Debian have only
+# version 0.9.0 or earlier OpenOCD packages (Ubuntu 17.04, Debian Jessie)
+# Set this to 1 if you are using one of these older releases.
+export USE_OLD_OPENOCD ?= 0
```

Comment:
Slightly out of the scope of this PR but I think the `export` is not needed, it's only used my make.

It would also be good to remove the equivalent line in the frdm-kw41z `Makefile.include`. Could be done in an extra commit.

Reply:
Will do 👍

Reply:
addressed